pages/billing/subscription.en-US.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Subscription
+ogImageTitle: Subscription
+ogImageSubtitle: Subscription on Zeabur
+---
+
+# Subscription
+
+## What is Subscription?
+
+There're 3 types of plans on Zeabur: **Free Plan**, **Developer Plan** and **Team Plan**.
+You can check the details of each plan on [plans](/billing/plans) page.
+
+To use Zeabur, you need to subscribe to one of the plans, the default plan is **Free Plan**.
+You can deploy your application with **Free Plan**, but there will be some limitations.
+
+## How to Subscribe?
+
+Zeabur's subscription is based on [Stripe](https://stripe.com/).
+
+There're 2 ways to subscribe to a plan: **Credit Card** and **Alipay**.
+
+### Subscribe with Credit Card
+
+You can subscribe to a plan with your credit card, paying with USD.
```

Comment:
`paying in USD`. [ref](https://www.bill.com/blog/pay-us-dollars-or-foreign-currency)